tests/conftest.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import pytest
+from dotenv import load_dotenv
+from rdflib import Graph, URIRef
+
+from pyrdfstore.build import create_rdf_store
+from pyrdfstore.store import MemoryRDFStore, URIRDFStore
+
+load_dotenv()
+
+
+@pytest.fixture()
+def rdf_store():
+    read_uri = os.getenv("TEST_SPARQL_READ_URI", None)
+    write_uri = os.getenv("TEST_SPARQL_WRITE_URI", None)
+    print(f"read_uri: {read_uri}")
+    print(f"write_uri: {write_uri}")
+    return create_rdf_store(read_uri, write_uri)
+    if read_uri is not None:
```

Comment:
lines 20 to 23 are unreachable and make no more sense -- @cedricdcc please remove this historic glitch before merge